zuul-core/src/main/java/com/netflix/zuul/netty/connectionpool/DefaultOriginChannelInitializer.java

```diff
@@ -82,6 +93,14 @@ protected void initChannel(Channel ch) throws Exception {
     protected void addMethodBindingHandler(ChannelPipeline pipeline) {
     }
 
+    private SslContext getClientSslContext() throws SSLException {
+        return SslContextBuilder
+                    .forClient()
+                    .sslProvider(BaseSslContextFactory.chooseSslProvider())
+                    .startTls(true)
```

Comment:
Why enable the `startTls` flag?

Reply:
this triggers the ssl handshake for the connection